ios/fastlane/verify/validate_metadata.rb

```
@@ -0,0 +1,103 @@
+#!/usr/bin/env ruby
```

RuboCop (1.85.1)

[warning] Script file validate_metadata.rb doesn't have execute permission.

(Lint/ScriptPermission)